view/frontend/templates/displayqr.phtml

```diff
@@ -0,0 +1,5 @@
+<?php $image = "data:".$block->getQrDataType().";base64, ".$block->getOfflineCode(); ?>
+<p><?= $block->escapeHtml(__('Amount to pay')) ?>: <strong><?= $block->getOrderAmount() ?></b></strong>
+<p style="margin:.5rem 0"><?= __('PromptPay QR Code.'); ?></p>
+<div><img id="img_payment_code" src="<?= $image ?>" /></div>
+<p><button class="action secondary" onclick="window.print ? window.print() : alert('Print not available')"><?= __('Print')?></button></p>
```

Comment:
This file's name suggests for the generic name of display QR code but it seems the code is more for the PromptPay. Should we make the name more specific toward PromptPay?

Reply:
Updated name of file to be more generic as we may plan to use this for similiar features @ 81b078e0b967d16b53bfb532673fd627c51c857f